files/anope/services.conf.j2

```diff
@@ -1,21 +1,21 @@
 define
 {
 	name = "services.host"
-	value = "services.irc.hashbang.sh"
+	value = "services.irc.dannygrove.com"
```

Comment:
i think we should define the top level domain (and possibly subdomains) as a variable so we don't have this issue per-deployment.

Reply:
Definitely a good idea. This is because I'm testing a bunch of stuff. I'm also now moving towards helm for this stuff, So most of this work is just to make sure i'm using the correct config style.